docs/examples/Ch-ExamplesAddingModbusDevice.rst

```diff
@@ -2,459 +2,273 @@
 
-  * Under “Key”, look for the drop down menu for “text”. Be sure to write “file” in the open box.
-  * Under “Value” click  “Choose Files”, locate your profile file.
 
-* Press Upload
-* Press the “Send” button when you are finished
-* Note the profile id
+Set Up Before Starting Services
+===============================
 
-**Post the device**
+The following sections describe how to complete the set up before starting the services. If you prefer to start the services and then add the device, see `Set Up After Starting Services`_
 
-* In the “Collections” tab select the option “POST /device
-* Click on the “Body” tab
-* Modify its contents
+Set Up Device Profile
+---------------------
 
-  * There are three components that are required to be modified. They are:
+The DeviceProfile defines the device's values and operation method, which can be Read or Write. 
 
-    * “Service”
-    * “Profile”
-    * “Addressable”
-    * The others can be modified, however they are not required for operation
+In the Modbus protocol, we must define attributes: 
 
-  * name: mbus-rth-lcd-device
-  * description: modbus-thermostat
-  * addressable:
+* ``primaryTable``: HOLDING_REGISTERS, INPUT_REGISTERS, COILS, DISCRETES_INPUT
+* ``startingAddress`` specifies the address in Modbus device
 
-    * name: mbus-rth-lcd-address (same as used in addressable)
-    * labels: “temperature”, “modbus”,”industrial” (same as used in modbus device profile)
+    .. image:: attributes.png
+        :scale: 50%
+        :alt: DeviceProfile Attributes
 
-  * service:
+The Property value type decides how many registers will be read. Like Holding registers, a register has 16 bits. If the device manual specifies that a value has two registers, define it as FLOAT32 or INT32 or UINT32 in the deviceProfile.
 
-    * name: edgex-device-modbus
+Once we execute a command, device-modbus knows its value type and register type, startingAddress, and register length. So it can read or write value using the modbus protocol.
 
-  * profile:
+    .. image:: properties.png
+        :scale: 50%
+        :alt: Properties
 
-    * name: mbus-rth-lcd (same as used in modbus device profile)
+|
+|
 
-* Press the “Send” button when you are finished
-* Note the addressable id
+    .. image:: holdingregisters.png
+        :scale: 70%
+        :alt: Holding Registers
 
-**What if a Mistake is Made**
+|
+|
 
-* Get device id
-* Delete device id
-* Get device profile id
-* Delete device profile id
-* Get addressable id
-* Delete addressable id
+    .. image:: profileyaml.png
+        :scale: 70%
+        :alt: Profile YAML
 
-**Verify Device Added**
 
-Check the edgex-device-modbus logs to see if the device was added without issue
+Set Up Device Service Configuration
+-----------------------------------
 
-“sudo docker logs -f --tail 100 edgex-device-modbus”
+Use this configuration file to define devices and schedule jobs. The device-modbus generates a relative instance on startup.
```

Comment:
Again, please make the configuration file downloadable or link to it on Github

Reply:
Updated